### DISCUSSION TITLE
Benchmark tool doesn't work based on documentation

### DISCUSSION THREAD

**Discussion by benjaminclauss** [etcd-io/etcd#16996] [Q&A] [answered]
- https://etcd.io/docs/v3.4/op-guide/performance/#benchmarks
- https://github.com/etcd-io/etcd/tree/v3.4.16/tools/benchmark

```
➜  ~ go get go.etcd.io/etcd/tools/benchmark
go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
```

**Comment by linsite** [comment]
I've also encountered this. Are there any new updates?

**Reply by jmhbnz** [reply, marked as answer]
For supporting tools like benchmark utility please refer to documentation in the `main` branch https://github.com/etcd-io/etcd/tree/main/tools/benchmark

> Install the tool by running the following command from the etcd source directory.

```  
$ go install -v ./tools/benchmark
```

**Reply by benjaminclauss** [reply]
looks like this documentation is now updated
